frame/gilt/src/lib.rs

```diff
@@ -128,7 +128,6 @@ pub mod pallet {
 
 		/// The issuance to ignore. This is subtracted from the `Currency`'s `total_issuance` to get
 		/// the issuance by which we inflate or deflate the gilt.
-		#[pallet::constant]
```

Comment:
We should probably add a note here so it is not accidentally added back again.

Reply:
I mean the description above already says that this value increases/decreases.